aioshelly/const.py

```diff
@@ -988,6 +989,13 @@ class ShellyDevice:
         gen=GEN3,
         supported=True,
     ),
+    MODEL_MULTICOLOR_BULB_G3: ShellyDevice(
+        model=MODEL_MULTICOLOR_BULB_G3,
+        name="Shelly Multicolor Bulb Gen3",
```

Comment:
The device presents itself in the user interface as `Multicolor Bulb Gen3`, the same name used on the Shelly community.